packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -556,15 +556,20 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     feeQuotesOrData: FeeQuotesOrDataDto,
   ): Promise<FeeQuotesOrDataResponse> {
     const paymaster = this.paymaster as IHybridPaymaster<PaymasterUserOperationDto>;
-    return paymaster.getPaymasterFeeQuotesOrData(userOp, feeQuotesOrData);
+    const tokenList = feeQuotesOrData?.preferredToken
```

Comment:
if preferred token is available actually you can just send preferred token afaik